BigCommerce/Integration/03_ManageBigCommerceSettings.md

```diff
@@ -0,0 +1,35 @@
+# Manage BigCommerce settings
+
+
+## Manage API settings
+
+As the driver creates automatically the necessary API settings during creating the connection, you have no need for further actions. See the *Store-level API account* in your *BigCommerce* store under *Settings > API*. 
+
+![Store-level API accounts](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/StoreLevelApiAccount.png "[BigCommerce settings]")
+
+
+## Manage multiple shipping addresses
+
+Although *BigCommerce* allows multiple shipping addresses, *Actindo* supports the import of one shipping address only. Any orders that use multiple shipping addresses cannot be imported and will fail during the import.  
+To avoid failures during the import, you must check a specific setting in *BigCommerce*. 
+
+#### Prerequisites
+
+- A *BigCommerce* connection has been created.
+- You have a valid user account in *BigCommerce*.
+
+#### Procedure
+
+*BigCommerce store > Settings*
+
+![BigCommerce settings](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/BigCommerceSettings.png "[BigCommerce settings]")
+
+1. Display the *Advanced* section and click the *Checkout* entry.
```

Comment:
Hier fehlen die Results, also "unsere" übliche Struktur ist etwas anders. Funktioniert mit:
1. Go to the *Advance* section and click the *Checkout* entry.
    The *Optimized one-page checkout settings* is displayed (oder was auch immer angezeigt wird).
    
    Screenshot

2. Go to ... and click ... 
    Result

   Screenshot

...